src/measure_theory/bochner_integration.lean

```diff
@@ -940,10 +950,10 @@ variables [normed_group E] [second_countable_topology E] [normed_space ℝ E] [c
 def integral (μ : measure α) (f : α → E) : E :=
 if hf : integrable f μ then (l1.of_fun f hf).integral else 0
 
-notation `∫` binders `, ` r:(scoped:60 f, f) ` ∂` μ:70 := integral μ r
-notation `∫` binders `, ` r:(scoped:60 f, integral volume f) := r
-notation `∫` binders ` in ` s `, ` r:(scoped:60 f, f) ` ∂` μ:70 := integral (measure.restrict μ s) r
-notation `∫` binders ` in ` s `, ` r:(scoped:60 f, integral (measure.restrict volume s) f) := r
+notation `∫` binders `, ` r:(scoped:0 f, f) ` ∂` μ:70 := integral μ r
+notation `∫` binders `, ` r:(scoped:0 f, integral volume f) := r
```

Comment:
How does it parse `∫ x, f x = 0`? I know that Lean fails to parse `∑ k, f k = 0` correctly.

Reply:
I think you made a typo in your question. The notation `∫ x, f x ∂μ = 0` is parsed fine, because the `∂μ` ends the integral (and the right binding power of the argument `μ` remains the same. This is also clear by the fact that mathlib compiles.

Reply:
This was the notation for `μ = volume`, not a typo.

Reply:
Oh I see. I forgot that we also have notation for that (even though I changed the line where it got declared).

Yes, this change will make that fail. Unfortunately Lean also doesn't allow changing the binding power for only some of the integration notation, then the overloading doesn't work properly.

I didn't expect `∫ x, f x = 0` since all other binders (also big set and lattice operations) will read through the end of the expression, as you mention. It seems like we can also use the binding power of integrals for the other binders. We can probably parse `∫ x, g ∥x∥ ∂μ` correctly by changing the binding power of `∥`?

Shall I undo this change?